themes-default/slim/views/addShows_newShow.mako

```diff
@@ -212,13 +218,33 @@ window.app = new Vue({
                 }
 
                 // Converts select boxes to command separated values [js/blackwhite.js]
-                generateBlackWhiteList(); // eslint-disable-line no-undef
+                //generateBlackWhiteList(); // eslint-disable-line no-undef
 
                 formData = new FormData(this.$refs.addShowForm);
             }
 
             this.otherShows.forEach(nextShow => formData.append('other_shows', nextShow));
 
+            // Because we're using te toggle-button.vue component, we don't have valid form input's for these.
+            // Therefor we need to add these values manually to the form data.
+
+            formData.append('subtitles', subtitles ? '1' : '0');
+            formData.append('anime', anime ? '1' : '0');
+            formData.append('scene', scene ? '1' : '0');
+            formData.append('season_folders', seasonFolder ? '1' : '0');
```

Comment:
All of these can use `Number(value)` instead of the conditionals, however it **should** (need to check add_shows.py code) also be working with any truthy value, and I think supplying booleans is the best option in the long term.

Reply:
Yeah I'd also rather pass booleans. And have the backend translate (as late as possible). Which in the long term should be done by the ORM, when there is no other way supported by db engine.

Reply:
Changed them to Number(subtitles).. etc.